tuf/repository_lib.py

```diff
@@ -496,10 +505,18 @@ def _get_written_metadata(metadata_signable):
   """
 
   # Explicitly specify the JSON separators for Python 2 + 3 consistency.
-  written_metadata_content = \
-    json.dumps(metadata_signable, indent=1, separators=(',', ': '),
-               sort_keys=True).encode('utf-8')
-  
+  if tuf.conf.METADATA_FORMAT == 'json':
+    written_metadata_content = json.dumps(
+        metadata_signable, indent=1, separators=(',', ': '),
+        sort_keys=True).encode('utf-8')
+
+  elif tuf.conf.METADATA_FORMAT == 'der':
+    written_metadata_content = asn1_codec.convert_signed_metadata_to_der(
+        metadata_signable) # TODO: <~> CURRENTLY WORKING HERE
+  else:
+    raise tuf.Error('Unsupported metadata format in configuration. Unable to '
```

Comment:
Might be helpful to state exactly which configuration file.   tuf.conf.py?  Note: It's now named tuf.settings.py